scripts/version_maker.sh

```diff
@@ -18,63 +22,126 @@ function yaml() {
     }'
 }
 
-# find files ended with chartserviceversion.yaml 
-FIND_CMD=`find ./charts -type f -name "*basetemplate.yaml"`
+# This function checks whether the kind is CSV or CE
+checker(){
+    if [[ $1 == "ChaosExperiment" ]]; then
+        echo $metadata_version
+    elif [[ $1 == "ChartServiceVersion" ]]; then
+        echo $spec_version 
+    fi
+}
 
-# add all the files to an array called files
-files=$(echo $FIND_CMD | tr " " "\n")
+# This function takes the old version from the last commit 
+# and increment the existing version by one unit.
+versionInc(){
+    echo "version inc"
+    file=$1
+    
+    eval $(yaml_parser $file)
+    if [[ $? == 0 ]]; then
+        existing_version=`checker $kind`
+        echo "Existing version: $existing_version"
 
-# looping from all the files
-for file in $files
-do
-    # get the latest version from the template file
-    eval $(yaml $file)
+        # stores the last pushed committed file as temp.yaml in the root directory 
+        # and will be deleted after the job
+        temp_file=`git show $last_commit_hash:$1 >> temp.yaml`
```

Comment:
In which line the value for variable `last_commit_hash` assigned?

Reply:
I forgot to merge, It's in a different branch